CHANGELOG.md

```
@@ -394,7 +394,8 @@ and this project adheres to
 - ✨(onboarding) add activation code logic for launch #62
 - 💄(chat) add code highlighting for LLM responses #67
 
-[unreleased]: https://github.com/suitenumerique/conversations/compare/v0.0.19...main
+[unreleased]: https://github.com/suitenumerique/conversations/compare/v0.0.20...main
```

markdownlint-cli2 (0.23.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)